source/stylesheets/application.scss

```diff
@@ -1,17 +1,66 @@
+//= require_directory ./components
```

Comment:
Where is that components directory? Sprockets gives me an error when I try to run your branch:
`require_directory argument must be a directory` because of this line!